src/sql/upgrade-81.sql

```diff
@@ -0,0 +1,5 @@
+alter table runcommandlogs add column uuid uuid;
+update runcommandlogs set uuid = gen_random_uuid() where uuid is null;
+alter table runcommandlogs alter column uuid set not null;
+alter table runcommandlogs add constraint RunCommandLogs_uuid_unique unique(uuid);
```

Comment:
I think the only real blocker at the moment is testing that this migration works as expected -- I added the unique constraint so that we're sure the UUID is indeed unique, but it's been a hot minute since I last used SQL. That's pretty much the only part I haven't tested.

Should be able to get to that on Monday.